modules/parsers/carousel.js

```diff
@@ -33,6 +33,9 @@ function getCommonAncestor(elements) {
 export default function parse(el) {
   // a carousel will consist of one row for every image found
   const images = el.querySelectorAll('img');
+  if (images.length === 1 && images[0].children.length === 0) {
+    return [[images[0]]];
+  }
```

Comment:
In Hubble's 'article' page, there was a carousel with only 1 image assigned to it.  There were no commonParent's children.